jsx/DataTable.js

```diff
@@ -435,6 +435,8 @@ class DataTable extends Component {
                     celldata,
                     row
                 );
+            } else {
+                cell = <td>{celldata}</td>;
             }
             if (cell !== null) {
```

Comment:
Line 441 will never be null.

Reply:
it can be null if getFormattedCell is defined and returns null

Reply:
@driusan oops my mistake.